docs/source/settingup.rst

```diff
@@ -1,16 +1,26 @@
 Setting up
 ==========
 
-Requirements:
+Requirements
+------------
 
 - DataLad
 
 Before being able to use the extension, you need to have DataLad installed, which relies on git-annex, git and python. If you don't have DataLad installed yet, please follow the instructions `here <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
 
+
 - An account on the OSF
 
 You need an OSF account to be able to interact with it. If you don't have an account yet, `register here <https://osf.io/register>`_.
 
 - An account on Github
 
-You need an account on Github to be able to interact with it and use git. If you don't have an account yet, `register here <https://github.com/join?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home>`_.
\ No newline at end of file
+You need an account on Github to be able to interact with it and use git. If you don't have an account yet, `register here <https://github.com/join?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home>`_.
+
+
+Getting ready
+-------------
+
+Before you can start using the extension, you have to install it. To do this, open your shell and type:
+
+``pip install -e [--user] git+https://github.com/datalad/datalad-osf#egg=datalad-osf``
```

Comment:
I would add these instructions as "developer installation".
After we cut a release, packed the software up for PyPi and uploaded it the installation is ``pip install datalad-osf`